FoodplannerServices/Account/ChildrenService.cs

```diff
@@ -24,19 +24,20 @@ public async Task<int> CreateChildrenAsync(ChildrenCreateParentDTO childrenCreat
     {
         var children = _mapper.Map<Children>(childrenCreateDTO);
         var childId = await _childrenRepository.InsertAsync(children);
-        
+
         foreach (var parentId in childrenCreateDTO.ParentIds)
         {
             await AddParentToChildAsync(parentId, childId);
         }
-        
+
         return childId;
     }
 
-    public async Task<IEnumerable<Children>> GetAllChildrenAsync()
+    public async Task<IEnumerable<ChildrenDTO>> GetAllChildrenAsync()
     {
         var children = await _childrenRepository.GetAllAsync();
-        return children;
+        var childrenDto = children.Select(child => _mapper.Map<ChildrenDTO>(child));
```

Comment:
I know i recommended this, but i think you can do this:
```suggestion
        var childrenDto = _mapper.Map<IEnumerable<ChildrenDTO>>(children);
```